src/stores/AppStore.js

```diff
@@ -440,7 +449,9 @@ export const AppStore = types
           self.project.num_tasks_with_annotations !== newProject.num_tasks_with_annotations
         ) : false;
 
-        if (JSON.stringify(newProject ?? {}) !== oldProject) {
+        if (options.interaction === "timer") {
+          self.project = Object.assign(oldProject, newProject);
```

Comment:
`oldProject` is a string here (which is not clear and a bit weird)

Reply:
That is right, good catch.